src/Physics/physicsHelper.ts

```diff
@@ -19,29 +19,37 @@ export class PhysicsHelper {
 
     private _scene: Scene;
     private _physicsEngine: Nullable<IPhysicsEngine>;
+    public options: PhysicsHelperOptions;
 
     /**
      * Initializes the Physics helper
      * @param scene Babylon.js scene
      */
-    constructor(scene: Scene) {
+    constructor(scene: Scene, options: PhysicsHelperOptions) {
         this._scene = scene;
         this._physicsEngine = this._scene.getPhysicsEngine();
 
         if (!this._physicsEngine) {
             Logger.Warn('Physics engine not enabled. Please enable the physics before you can use the methods.');
         }
+
+        this.options = {...(new PhysicsHelperOptions()), ...options};
+
+        if (this.options.useNativePhysicsRaycastingIfAvailable) {
+          if (this._physicsEngine.getPhysicsPluginName() === 'OimoJSPlugin') {
+            this.options.useNativePhysicsRaycastingIfAvailable = false;
+            Logger.Warn('Native raycasting not available on the OimoJSPlugin.');
+          }
+        }
     }
 
     /**
      * Applies a radial explosion impulse
      * @param origin the origin of the explosion
-     * @param radius the explosion radius
-     * @param strength the explosion strength
-     * @param falloff possible options: Constant & Linear. Defaults to Constant
+     * @param eventOptions the options of radial explosion
      * @returns A physics radial explosion event, or null
      */
-    public applyRadialExplosionImpulse(origin: Vector3, radius: number, strength: number, falloff: PhysicsRadialImpulseFalloff = PhysicsRadialImpulseFalloff.Constant): Nullable<PhysicsRadialExplosionEvent> {
+    public applyRadialExplosionImpulse(origin: Vector3, eventOptions: PhysicsRadialExplosionEventOptions): Nullable<PhysicsRadialExplosionEvent> {

```

Comment:
please mention that change in the breaking change

Reply:
This will also require a documentation update btw

Reply:
other option will be to NOT introduce breaking changes but instead provide a second signature:
public applyRadialExplosionImpulseEx(origin: Vector3, eventOptions: PhysicsRadialExplosionEventOptions):

and then have the first one calling that new one

Reply:
I did mention it here (https://github.com/BabylonJS/Babylon.js/pull/5935/files#diff-683d2ac1d77ed946cbfda3917f406c8dR264), but maybe I should be more specific.
Oh yeah, will update the documentation, once we are clear that everything else is fine.

I wasn't really sure if people even use this, so that's why I was somewhat not sure, if it was that important, if it had a breaking change - but yeah, I agree, should be avoided if possible...
What about if the second parameter (here and in all other calls is `radius: number | PhysicsRadialExplosionEventOptions` and create a private second function `applyRadialExplosionImpulseEx` - would that kinda fit into the context?

Reply:
yep I like that! it fixes the breaking change issue

Reply:
Ok, refactored that part!